src/components/clientInfo/ClientInfo.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import CarList from "./CarList";
+import CarDetails from "./CarDetails";
+import {useParams} from "react-router-dom";
+
+export default function ClientInfo(props) {
+    let params = useParams();
+    const clientName = "Last Name, First Name";
+    const currentCarID = "10001";
```

Comment:
We should probably change this hardcoded data into inputs for the functions and classes

Reply:
Yes, that is going in the next PR just to keep changes small